src/__stories__/tooltip-story.tsx

```diff
@@ -62,12 +62,26 @@ export const Default: StoryComponent = () => {
                 />
                 <Tooltip
                     targetLabel="help text"
-                    target={<Text2 regular>Tooltip with link</Text2>}
+                    target={
+                        <div style={{textAlign: 'center'}}>
+                            <Stack space={0}>
+                                <Text2 regular>Tooltip with custom elements</Text2>
+                                <Text1 regular color={colors.textSecondary}>
+                                    (do not use interactive elements inside)
+                                </Text1>
+                            </Stack>
+                        </div>
+                    }
                     position="top"
                     description={description}
-                >
-                    <a href="#whatever">Ir a la web</a>
-                </Tooltip>
+                    extra={<Placeholder />}
+                />
+                <Tooltip
+                    targetLabel="help text"
+                    target={<Text2 regular>Tooltip with delay</Text2>}
```

Comment:
does this example makes sense? all the Tooltips have delay now

Reply:
changed